testing/trino-product-tests-launcher/src/main/resources/docker/presto-product-tests/conf/environment/singlenode-kafka-ssl/secrets/create-certs.sh

```diff
@@ -0,0 +1,46 @@
+#!/bin/bash
+
+set -o nounset \
```

Comment:
we typically are setting `set -xeuo pipefail`. Please follow convention. Is your comment better? Should we change convention?

Reply:
Hi @kokosing , this script was taken from [confluent repo](https://github.com/confluentinc/kafka-images/blob/master/examples/kafka-cluster-ssl/secrets/create-certs.sh). TBH, I have not researched thoroughly those flags from Confluent's script. For now, I have set it to what you recommended instead.